### PR TITLE
[TE] frontend - Minor ui fixes

### PR DESCRIPTION
**Pull request by ttbach**
1. Fix metric cards alignment
2. Moved metrics tab to first 
3. Changed "card" to default view

screen shot 2018-01-12 at 11 37 12 am

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default Ember.Controller.extend({
       invisibleUrns: new Set(),
       hoverUrns: new Set(),
       filteredUrns: new Set(),
-      activeTab: ROOTCAUSE_TAB_DIMENSIONS,
+      activeTab: ROOTCAUSE_TAB_METRICS,
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: Is this intentional - metrics as the default view? afaik the heatmap should be the first thing

File: thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs [hunk 2]
```diff
@@ -97,13 +97,14 @@
...
+          <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "metrics") "thirdeye-link--active "}}" {{action (mut activeTab)
+            "metrics"}}>
+            Metrics
           </a>
         </div>
         <div class="card-container__subnav">
-          <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "metrics") "thirdeye-link--active"}}" {{action (mut activeTab) "metrics"}}>
-            Metrics
+          <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "dimensions") "thirdeye-link--active"}}" {{action (mut activeTab) "dimensions"}}>
+            Dimensions
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs]: same

File: thirdeye/thirdeye-frontend/app/styles/components/rootcause-metric.scss [hunk 3]
```diff
@@ -31,6 +31,7 @@
   }
 
   &__card {
+    height: 130px;
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/styles/components/rootcause-metric.scss]: did you try this with metrics with very long names, e.g. ones related to page views?

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js (hunk 1 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: Selene suggested to move the metrics tab to the first tab (see task spreadsheet). If it's the first tab, then it should be the default one that's shown.

File: thirdeye/thirdeye-frontend/app/styles/components/rootcause-metric.scss (hunk 3 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/styles/components/rootcause-metric.scss]: Yes :)

**Pull request merged**